dlrs/main/lwc/rollupEditor/rollupEditor.html

```diff
@@ -1,269 +1,542 @@
+                  </div>
+
+                  <!-- Aggregate Options -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-combobox
+                      name="rollup_AggregateOperation__c"
+                      data-name="rollup_AggregateOperation__c"
+                      label="Aggregate Operation"
+                      placeholder="Select"
+                      options={aggregateOptions}
+                      value={rollup.AggregateOperation__c}
+                      required
+                    ></lightning-combobox>
+                    <c-rollup-editor-error
+                      errors={errors.FieldToOrderBy__c}
+                    ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Aggregate Result Field -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-input
+                      type="text"
+                      name="rollup_AggregateResultField__c"
+                      data-name="rollup_AggregateResultField__c"
+                      label="Aggregate Result Field"
+                      value={rollup.AggregateResultField__c}
+                    ></lightning-input>
+                    <c-rollup-editor-error
+                      errors={errors.AggregateResultField__c}
+                    ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Aggregate All Rows -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-input
+                      type="checkbox"
+                      name="rollup_AggregateAllRows__c"
+                      data-name="rollup_AggregateAllRows__c"
+                      label="Aggregate All Rows"
+                      value={rollup.AggregateAllRows__c}
+                      field-level-help="Include deleted records"
+                      variant="label-stacked"
+                      style="padding-left: 0px !important"
+                    ></lightning-input>
+                    <c-rollup-editor-error
+                      errors={errors.AggregateAllRows__c}
+                    ></c-rollup-editor-error>
+                  </div>
+
+                  <!-- Row Limit -->
+                  <div class="slds-m-top_x-small">
+                    <lightning-input
+                      type="number"
+                      name="rollup_RowLimit__c"
+                      data-name="rollup_RowLimit__c"
+                      label="Row Limit"
+                      value={rollup.RowLimit__c}
+                    ></lightning-input>
+                    <c-rollup-editor-error
+                      errors={errors.RowLimit__c}
+                    ></c-rollup-editor-error>
+                  </div>
+                </div>
+
+                <div class="slds-col slds-size_1-of-1 slds-medium-size_1-of-2">
+                    <!-- Active Toggle -->
+                    <div class="slds-m-top_x-small">
+                        <lightning-input
```

Comment:
I don't think we want to expose the `Active__c` value here. For this wizard we're going to move it to a post-save button. This is largely because we have many people get confused when they try and set a new Rollup to Active but it complains about the missing trigger so doesn't let them save.
Having it as a post-save action lets us clear that UX phase.

Reply:
Yeah, I think so. `Active__c` value should be it to post-save. But you see we have a lot of buttons. I suggest the button group should be responsive like below:

![image](https://user-images.githubusercontent.com/64265243/231627555-0a4676ac-646e-4f42-bd96-ef5b0d923656.png)